plugins/node/opentelemetry-instrumentation-hapi/package.json

```diff
@@ -43,13 +43,14 @@
     "@opentelemetry/api": "^1.3.0"
   },
   "devDependencies": {
-    "@hapi/hapi": "20.1.5",
+    "@hapi/hapi": "21.3.3",
     "@opentelemetry/api": "^1.3.0",
     "@opentelemetry/context-async-hooks": "^1.8.0",
     "@opentelemetry/sdk-trace-base": "^1.8.0",
     "@opentelemetry/sdk-trace-node": "^1.8.0",
     "@types/mocha": "7.0.2",
     "@types/node": "18.6.5",
+    "joi": "17.12.2",
```

Comment:
This TypeScript project is setup to do lib checks. The alternative is to add `skipLibCheck: true` to `tsconfig.json`

https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1491#issuecomment-1967943017

Reply:
Judging by the rest of the code base, adding `skipLibCheck: true` seems like the right thing to do.

Change is pushed.

Reply:
While this worked for the hapi plugin itself, it causes the `auto-instrumentations-node` build to fail, presumably because that is now doing a transitive lib check. I'll bring back the joi dev-dependency, but I wonder if there are any other ideas on how to solve this?